kernel/libs/aster_page_allocator/src/lib.rs

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! This crate provides a global page allocator for the kernel.
+//!
+//! Currently, the page allocator is implemented by the buddy system.
+//! By corporating with the `PageAlloc` trait and `page_allocator_init`
+//! procedure macro, page allocator's implementation can be decoupled from the
+//! ostd and can be easily replaced to serve desinated purposes.
```

Comment:
`[OSTD](https://crates.io/crates/ostd)`

And there's a typo "desinated". Please double check if the typo checker can cover this crate.

Reply:
Typo checker can cover this crate, but the typo "desinated" cannot be detected. Probably because the typo checker's dictionary is imperfect.